src/date-range-picker/calendar/header/index.tsx

```diff
@@ -23,15 +24,16 @@ export default function CalendarHeader({
   previousMonthLabel,
   nextMonthLabel,
   isSingleGrid,
+  headingId,
 }: CalendarHeaderProps) {
   const prevMonthLabel = renderMonthAndYear(locale, add(baseDate, { months: -1 }));
   const currentMonthLabel = renderMonthAndYear(locale, baseDate);
 
   return (
     <>
-      <div className={styles['calendar-header']}>
+      <div role="heading" aria-level={2} className={styles['calendar-header']}>
```

Comment:
Only months should be heading, not the button?

Reply:
Thanks! Updated in the new revision